MonticelloTonel-Core.package/TonelRepository.class/instance/hash.st

```diff
@@ -0,0 +1,4 @@
+accessing
```

Comment:
Doesn't #hash usually belong into the 'comparing' protocol? In Object it is in 'comparing'.